SpaceCadetPinball/control.cpp

```diff
@@ -4568,7 +4632,11 @@ void control::TimeWarpPartTwoController(MessageCode code, TPinballComponent* cal
 	lite304->Message(MessageCode::TLightResetAndTurnOff, 0.0);
 	lite317->Message(MessageCode::TLightResetAndTurnOff, 0.0);
 	lite198->MessageField = 1;
-	MissionControl(MessageCode::ControlMissionComplete, nullptr);
+	if (pb::FullTiltMode)
+	{
+		table_set_replay(4.0);
+	}
+MissionControl(MessageCode::ControlMissionComplete, nullptr);
```

Comment:
and another mysterious case of too little indentation. If these look fine in your IDE, check how it looks in the github diff.